Minizip.podspec

```diff
@@ -14,24 +14,42 @@ DESC
   s.authors = 'Nathan Moinvaziri', 'Gilles Vollant'
 
   s.source   = { :git => 'https://github.com/nmoinvaz/minizip.git' }
-  s.libraries = 'z'
+  s.libraries = 'z', 'iconv'
+  s.default_subspecs = 'Core', 'PKWARE', 'crypt_apple', 'BZIP2'
 
   s.subspec 'Core' do |sp|
-    sp.source_files = '{mz_os,mz_compat,mz_strm,mz_strm_mem,mz_strm_buf,mz_zip,mz_strm_crypt,mz_strm_posix,mz_strm_zlib}.{c,h}'
+    sp.source_files = '{mz,mz_os,mz_os_posix,mz_compat,mz_crypt,mz_strm,mz_strm_mem,mz_strm_buf,mz_strm_crypt,mz_strm_os_posix,mz_strm_zlib,mz_zip}.{c,h}'
+    sp.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => 'HAVE_INTTYPES_H HAVE_STDINT_H HAVE_ZLIB' }
   end
 
-  s.subspec 'AES' do |sp|
+  s.subspec 'PKWARE' do |sp|
     sp.dependency 'Minizip/Core'
-    sp.source_files = 'lib/aes/*.{c,h}', 'mz_strm_aes.{c,h}'
+    sp.source_files = 'mz_strm_pkcrypt.{c,h}'
+    sp.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => 'HAVE_PKCRYPT' }
+  end
+
+  s.subspec 'crypt_apple' do |sp|
```

Comment:
I think I'd perfer the names of the submodules to closely match the names of the CMAKE options. Maybe something like this, if it is doable? 

PKWARE -> 'PKCRYPT'
crypt_apple -> 'WZAES - CommonCrypto'
crypt_brg -> 'WZAES - BRG'

What do you think?

Reply:
OK for WZAES_* and PKCRYPT. But for Core, I'll leave it capitalized, as we normally don't uppercase spec names anyway.
Don't worry for maintaining, when people will use it (maybe ZipArchive), we'll pull request if needed.